sktime/transformations/series/tests/date.py

```diff
@@ -0,0 +1,335 @@
+#!/usr/bin/env python3 -u
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Extract calendar features from datetimeindex."""
+__author__ = ["danbartl"]
+__all__ = ["DateTimeFeatures"]
+
+import warnings
+
+import numpy as np
+import pandas as pd
+
+from sktime.transformations.base import BaseTransformer
+
+_RAW_DUMMIES = [
```

Comment:
Out of curiosity, is there any reason `_RAW_DUMMIES` , `DateTimeFeatures` and others are redefined here instead of importing them from `sktime.transformations.series.date`

Reply:
Hi, thanks for the feedback! sorry I do not understand your point, sktime.transformations.series.date defines DateTimeFeatures.

Reply:
[sktime.transformations.series.date](https://github.com/alan-turing-institute/sktime/blob/main/sktime/transformations/series/date.py) also defines `_RAW_DUMMIES` similar to this variable.

Reply:
oh sorry I think I moved the file to the wrong location, thanks for pointing that out